packages/elements/src/tree/elements/tree-item.ts

```diff
@@ -156,24 +157,44 @@ export class TreeItem<T extends TreeDataItem = TreeDataItem> extends ControlElem
     return this.checkedState === CheckedState.INDETERMINATE;
   }
 
+  /**
+   * Invoked whenever the element is updated
+   * @param {PropertyValues} changedProperties Map of changed properties with old values
+   * @returns {void}
+   */
   protected update (changedProperties: PropertyValues): void {
     super.update(changedProperties);
+
     if (changedProperties.has('checkedState')) {
+      // Parent node in single-mode cannot be selected
+      if (this.parent && !this.multiple) {
+        return;
+      }
```

Comment:
Elements can be reused, so I think it should still process and remove any attributes.

Reply:
Fixed.